src/bookItems.js

```diff
@@ -0,0 +1,12 @@
+const bookItems = [];
+
+//   {
+//     id: '',
+//     genre: '',
+//     title: '',
+//     author: '',
+//     percentageCompletion: '',
+//     completed: '',
+//   },
```

Comment:
- [OPTIONAL] I will highly recommend you to remove this commented code as it makes our codebase looks ugly and unprofessional.